frontend/components/surplus/SurplusAuctionBidTransactionTable.vue

```diff
@@ -0,0 +1,145 @@
+                        :disabled="!isActive"
+                        auction-type="surplus"
+                    />
+                </div>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Combined Transaction Fees</div>
+            <div>
+                <FormatCurrency v-if="auction.combinedBidFees" :value="auction.combinedBidFees" currency="DAI" />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Price on Uniswap</div>
+            <div>
+                <format-currency
+                    v-if="auction.marketUnitPrice && isActive"
+                    :value="auction.marketUnitPrice"
+                    currency="MKR"
+                />
+                <span v-else class="opacity-50">Unknown</span> per DAI
+            </div>
+        </div>
+        <div class="flex justify-between font-bold">
+            <div>Price after the bid</div>
+            <div>
+                <format-currency
+                    v-if="unitPriceAfterBid && isActive && !isBidAmountNaN"
+                    :value="unitPriceAfterBid"
+                    currency="MKR"
+                />
+                <span v-else class="opacity-50">Unknown</span> per DAI
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import type { SurplusAuction } from 'auctions-core/src/types';
+import Vue from 'vue';
+import BigNumber from 'bignumber.js';
+import BidInput from '../utils/BidInput.vue';
+import TimeTill from '~/components/common/TimeTill.vue';
+import FormatCurrency from '~/components/utils/FormatCurrency.vue';
+
+export default Vue.extend({
+    components: {
+        TimeTill,
+        FormatCurrency,
+        BidInput,
+    },
+    props: {
+        auction: {
+            type: Object as Vue.PropType<SurplusAuction>,
+            required: true,
+        },
+        lowestNextBid: {
+            type: Object as Vue.PropType<BigNumber>,
+            required: true,
+        },
+    },
+    data() {
+        return {
+            transactionBidAmount: undefined as BigNumber | undefined,
+        };
+    },
+    computed: {
+        isActive(): boolean {
+            return this.auction.state === 'just-started' || this.auction.state === 'have-bids';
+        },
+        isBidAmountNaN(): boolean {
+            return !!this.transactionBidAmount?.isNaN();
+        },
+        unitPriceAfterBid(): BigNumber | undefined {
+            if (!this.transactionBidAmount) {
+                return this.lowestNextBid.dividedBy(this.auction.receiveAmountDAI || 1);
+            }
+            return this.transactionBidAmount.dividedBy(this.auction.receiveAmountDAI || 1);
+        },
+    },
+    watch: {
+        transactionBidAmount: {
+            immediate: true,
+            handler(transactionBidAmount) {
+                this.$emit('inputBidAmount', transactionBidAmount);
```

Comment:
Can those two things be called the same way? Or what is the difference between them?

Reply:
I am sorry, I don't quit understand the question 😓

Reply:
What is the difference between `transactionBidAmount` and `inputBidAmount` so they are called differently?

Reply:
I took this naming from the original component [here](https://github.com/sidestream-tech/unified-auctions-ui/blob/main/frontend/components/transaction/BidTransactionTable.vue#L148). I do not know what the logic behind it was when first written, however I wanted to keep the two somewhat similar.

Reply:
Renamed the transactionBidAmount to inputBidAmount.

Reply:
Please note that this name suppose to be the same as used in all other components that refer to the bidding amount. What is the name for it in the other places? `inputBidAmount` sounds local to this component (since this component is using `input` to get this value)

Reply:
True, then I will rename it just to bidAmount.